src/OpenTelemetry.Contrib.Instrumentation.Azure/Implementation/AzureSdkDiagnosticListener.cs

```diff
@@ -102,9 +102,7 @@ public override void OnStopActivity(Activity current, object valueValue)
 
         public override void OnException(Activity activity, object valueValue)
         {
-            Status status = Status.Unknown;
-            activity.AddTag(SpanAttributeConstants.StatusCodeKey, SpanHelper.GetCachedCanonicalCodeString(status.CanonicalCode));
-            activity.AddTag(SpanAttributeConstants.StatusDescriptionKey, valueValue?.ToString());
```

Comment:
I think we can maintain this line

Reply:
I've changed it to this code in order to save a description
```csharp
activity.SetStatus(Status.Error.WithDescription(valueValue?.ToString()))
```